distributed/client.py

```diff
@@ -5495,39 +5496,45 @@ async def __aexit__(self, exc_type, exc_value, traceback):
 class performance_report:
     """Gather performance report
 
-    This creates a static HTML file that includes many of the same plots of the
-    dashboard for later viewing.
+     This creates a static HTML file that includes many of the same plots of the
+     dashboard for later viewing.
 
-    The resulting file uses JavaScript, and so must be viewed with a web
-    browser.  Locally we recommend using ``python -m http.server`` or hosting
-    the file live online.
+     The resulting file uses JavaScript, and so must be viewed with a web
+     browser.  Locally we recommend using ``python -m http.server`` or hosting
+     the file live online.
 
-    Parameters
-    ----------
-    filename: str, optional
-        The filename to save the performance report locally
+     Parameters
+     ----------
+     filename: str, optional
+         The filename to save the performance report locally
 
-    stacklevel: int, optional
-        The code execution frame utilized for populating the Calling Code section
-        of the report. Defaults to `1` which is the frame calling ``performance_report``
+     stacklevel: int, optional
+         The code execution frame utilized for populating the Calling Code section
+         of the report. Defaults to `1` which is the frame calling ``performance_report``
 
-    mode: str, optional
-        Mode parameter to pass to :func:`bokeh.io.output.output_file`. Defaults to ``None``.
+     mode: str, optional
+         Mode parameter to pass to :func:`bokeh.io.output.output_file`. Defaults to ``None``.
 
-    Examples
-    --------
-    >>> with performance_report(filename="myfile.html", stacklevel=1):
-    ...     x.compute()
+    **storage_options:
+         Any additional arguments to :func:`fsspec.open` when writing to a URL.
+
+     Examples
+     --------
+     >>> with performance_report(filename="myfile.html", stacklevel=1):
+     ...     x.compute()
 
-    $ python -m http.server
-    $ open myfile.html
+     $ python -m http.server
+     $ open myfile.html
     """
 
-    def __init__(self, filename="dask-report.html", stacklevel=1, mode=None):
+    def __init__(
+        self, filename="dask-report.html", stacklevel=1, mode=None, **storage_options
```

Comment:
To avoid conflict with adding other keyword arguments in the future, let's have `storage_options` be passed in as a dict instead of via `**kwargs`. This will also match how we use `storage_options` in other places (e.g. `dask.dataframe.read_parquet`, `dask.array.to_zarr`, etc.) 

```suggestion
        self, filename="dask-report.html", stacklevel=1, mode=None, storage_options=None
```